spec/lib/oauth/token_spec.rb

```diff
@@ -96,13 +96,18 @@ module OAuth
       end
 
       describe :authenticate do
-        let(:finder) { double :finder }
-
-        it 'calls the finder if token was found' do
+        it 'calls the finder if token was returned' do
           token = ->(r) { 'token' }
           expect(AccessToken).to receive(:by_token).with('token')
           Token.authenticate double, token
         end
+
+        it 'revokes previous refresh_token if token was found' do
+          token = ->(r) { 'token' }
```

Comment:
Unused block argument - `r`. If it's necessary, use `_` or `_r` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.